### PR TITLE
Add aarch64 support to Dockerfile.worker.

### PR DESCRIPTION
**Pull request by tdaede**

### CODE REVIEW

File: Dockerfile.worker [hunk 1]
```diff
@@ -115,13 +132,21 @@ ENV \
...
 	cd ${HDRTOOLS_DIR} && \
 	sed -i 's/std::modff/modff/g' common/src/OutputY4M.cpp && \
 	sed -i 's/using ::hdrtoolslib::Y_COMP;//g' projects/HDRConvScaler/src/HDRConvScalerYUV.cpp && \
 	sed -i 's/\[Y_COMP\]/\[hdrtoolslib::Y_COMP\]/g' projects/HDRConvScaler/src/HDRConvScalerYUV.cpp && \
+	if [ "$ARCH" = "aarch64" ]; then \
+		sed -i 's/-msse2//g' common/Makefile projects/*/Makefile; \
+		sed -i 's/-mfpmath=sse//g' common/Makefile projects/*/Makefile; \
+    	sed -i 's/#include <x86intrin.h>//g' common/src/ResizeBiCubic.cpp common/src/DistortionMetricVQM.cpp; \
+        sed -i 's/#include <mmintrin.h>//g' common/src/DistortionMetricVQM.cpp; \
+		sed -i 's/#if defined(ENABLE_SSE_OPT)/#if ENABLE_SSE_OPT/g' common/src/ResizeBiCubic.cpp; \
```
Comment (tmatth) [inline Dockerfile.worker:148]: nit: weird mix of spaces and tabs heere, we should stick to tabs

Comment (tdaede) [reply Dockerfile.worker:148]: fixed

Comment (negge) [reply Dockerfile.worker:148]: Should these be upstreamed? I cannot imagine this is the only project that wants to run HDRTools on aarch64.

Comment (tdaede) [reply Dockerfile.worker:149]: yes, working on it

**Review by tmatth — APPROVED** [review]

**Comment by negge** [comment]
LGTM

**Pull request merged**